src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx

```diff
@@ -0,0 +1,160 @@
+            <li className="vads-u-padding-left--1">
+              When the app (or website) prompts you to connect your VA account,
+              it will ask you to sign in.
+            </li>
+            <li className="vads-u-padding-left--1">
+              Sign in with your preferred VA.gov account: DS Logon, My
+              HealtheVet, or ID.me.
+            </li>
+            <li className="vads-u-padding-left--1">
+              Review the information the app is asking to access.
+            </li>
+          </ol>
+          <p>
+            If you’re comfortable sharing that information, allow access. If you
+            have trouble connecting the app, contact the app’s support for help.
+          </p>
+        </AdditionalInfo>
+      </div>
+
+      {!isEmpty(activeApps) && (
+        <div className="vads-u-margin-bottom--2">
+          <AdditionalInfo
+            triggerText={`What should I do if my records are wrong or out of date in a connected app?`}
+          >
+            <p>
+              <strong>This depends on the issue:</strong>
+            </p>
+            <ul className="vads-u-margin-left--3">
+              <li className="vads-u-padding-left--1">
+                <strong>If your health records are missing:</strong> It can take
+                up to 3 days for new health records to show in a connected app.
+                If it’s been more than 3 days since your last appointment, or if
+                you need your information sooner,{' '}
+                <a
+                  href="https://www.myhealth.va.gov/mhv-portal-web/user-login?redirect=/mhv-portal-web/home"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  sign in to My HealtheVet
+                </a>{' '}
+                or contact your VA health care team.
+              </li>
+              <li className="vads-u-padding-left--1">
+                <strong>If your information isn’t accurate:</strong> Call VA311
+                at <Telephone contact={CONTACTS.VA_311} />. If you have hearing
+                loss, call TTY: <a href="tel:711">711</a>. Or visit a VA health
+                facility near you and ask a staff member for help.{' '}
+                <p>
+                  <a href="/find-locations">
+                    Find a VA health facility near you
+                  </a>
+                </p>
+              </li>
+              <li className="vads-u-padding-left--1">
+                <strong>If you’re getting an “unreadable data” message:</strong>{' '}
+                This means the This means the connected app has access to your
+                information, but isn’t using it in its interface. It’s nothing
+                to worry about. If you have questions about this, send feedback
+                directly to the app.
+              </li>
+            </ul>
+          </AdditionalInfo>
+        </div>
+      )}
+
+      {!isEmpty(activeApps) && (
```

Comment:
oh and it looks like this conditional is the same as line 85 so these can be combined?